### PR TITLE
Text encoding

### PR DESCRIPTION
**Pull request by g4s8**
Now we are able to use charset to convert bytes to text and vice verse.
Default (`Charset.defaultCharset()`) charset is used in secondary constructors.
Part of #13

### CODE REVIEW

**Comment by 0crat** [comment]
@g4s8 I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by yegor256** [comment]
@g4s8 we seem to have problem. The encoding inside Docker container is _not_ UTF-8.

**Comment by yegor256** [comment]
@g4s8 I think it's better to fix our code and leave the non-UTF-8 encoding inside Docker, for more fragility of future tests.

**Comment by g4s8** [comment]
@yegor256  agree, I'm pulling this docker image `yegor256/java8` to test it locally

**Comment by g4s8** [comment]
@yegor256 I've added "UTF-8" charset argument for failing tests.

**Comment by yegor256** [comment]
@g4s8 why not this: https://docs.oracle.com/javase/7/docs/api/java/nio/charset/StandardCharsets.html#UTF_8

**Comment by g4s8** [comment]
@yegor256 sorry, forget about them. Replaced with `StandardCharsets` now

File: src/test/java/org/cactoos/io/FileAsInputTest.java [hunk 1]
```diff
@@ -44,14 +46,16 @@
      */
     @Test
     public void readsFileContent() throws IOException {
+        final Charset utf = StandardCharsets.UTF_8;
```
Comment (yegor256) [inline src/test/java/org/cactoos/io/FileAsInputTest.java]: @g4s8 just use it inline. What's the point of a new variable? Remember, we're trying to keep our tests statement-free, as this article recommends: http://www.yegor256.com/2017/05/17/single-statement-unit-tests.html

**Comment by yegor256** [comment]
@g4s8 one more comment above

File: src/test/java/org/cactoos/io/FileAsInputTest.java (hunk 1 above)
Comment (g4s8) [inline src/test/java/org/cactoos/io/FileAsInputTest.java]: Done!

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by yegor256** [comment]
@g4s8 one more problem is still there

**Comment by g4s8** [comment]
@yegor256 added missed charset

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by yegor256** [comment]
@g4s8 many thanks!

**Pull request merged**